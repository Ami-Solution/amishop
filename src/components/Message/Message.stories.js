/* eslint-disable */
import React from 'react'
import styled, { css } from 'styled-components'
import { storiesOf } from '@storybook/react'

import Message from './index'

storiesOf('03 - Components', module)
.addWithChapters('Messages', {
  subtitle: 'Message error',
  chapters: [
    {
      sections: [
      {
        title: 'Select Input',
        sectionFn: () => (
          <Message theme="error">
            Hi AmiShop here; this is an error Message. Lieber Amicorum..
          </Message>
        ),
        options: {
          showSource: true,
          showPropTables: true
        }
      },{
        title: 'Select Input',
        sectionFn: () => (
          <Message theme="info">
            Hi AmiShop here; this is an informational Message. Lieber Amicorum..
          </Message>
        ),
        options: {
          showSource: true,
          showPropTables: true
        }
      },{
        title: 'Select Input',
        sectionFn: () => (
          <Message withIcon={false} >
            Hi AmiShop here; this is a Message without icon. Lieber Amicorum..
          </Message>
        ),
        options: {
          showSource: true,
          showPropTables: true
        }
      }
    ]
  }]
});
