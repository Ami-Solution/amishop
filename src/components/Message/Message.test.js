/* global describe it expect */
import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import Message from './index';

configure({ adapter: new Adapter() });

describe('Component Message', () => {
  const component = renderer
    .create(
      <Message theme="info">
        Hello AmiShop; this is a test Message. Lieber Amicorum...
      </Message>
    );
  const tree = component.toJSON();

  it('should be render without crash', () => {
    expect(tree).toMatchSnapshot();
  });
});
