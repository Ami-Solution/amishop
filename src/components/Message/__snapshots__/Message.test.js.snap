// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Message should be render without crash 1`] = `
<div
  className="sc-bdVaJa kvKDTO"
>
  <div
    className="sc-bwzfXH gKlmNv"
  />
  <div>
    Hello AmiShop here; this is a test Message. Lieber Amicorum..
  </div>
</div>
`;
